include/sway/container.h

```diff
@@ -162,4 +162,10 @@ void container_map(swayc_t *container,
 swayc_t *swayc_at(swayc_t *parent, double lx, double ly,
 		struct wlr_surface **surface, double *sx, double *sy);
 
+/**
+ * Get a list of containers that are descendents of the container in rendering
+ * order
```

Comment:
Can you define "rendering order" for me

Reply:
"rendering order" means the order in which containers go through the rendering loop. Prior elements in rendering order are rendered "on top of" later elements. Parent containers are rendered below their children, so they will be earlier in the list. Floating containers appear on top of tiled containers, so they go in the list earlier.

For a given cursor coordinate, the cursor position might be under several containers at the same time, such as when the cursor is over a floating window with a tiled window underneath it. To determine which window gets the mouse click, we find the first window in "reverse rendering order". This is the window the user sees on top of the others.

I'm sort of inventing this vocabulary right now, but I think it fits the domain.